src/components/nve-label/nve-label.demo.ts

```diff
@@ -74,18 +74,40 @@ const table = html`
         </td>
       </tr>
       <tr>
-        <td>Innhold i slot</td>
+        <td>Med info-ikon på venstre siden</td>
```

Comment:
Hva prøver du å vise her?
Er det plassering av info-ikon eller er det slotta innhold?

Reply:
Tooltip (info-ikon) er eneste ikon som vises i nve-label så jeg trenger å bruke tooltip slot for å vise ikonen